cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/bridge_to_bulletin_config.rs

```diff
@@ -153,31 +112,46 @@ impl pallet_bridge_messages::Config<WithRococoBulletinMessagesInstance> for Runt
 	type BridgedChain = bp_polkadot_bulletin::PolkadotBulletin;
 	type BridgedHeaderChain = BridgeRococoBulletinGrandpa;
 
-	type ActiveOutboundLanes = ActiveOutboundLanesToRococoBulletin;
-
 	type OutboundPayload = XcmAsPlainPayload;
 
 	type InboundPayload = XcmAsPlainPayload;
 	type DeliveryPayments = ();
 
 	type DeliveryConfirmationPayments = ();
 
-	type MessageDispatch =
-		XcmBlobMessageDispatch<FromRococoBulletinMessageBlobDispatcher, Self::WeightInfo, ()>;
-	type OnMessagesDelivered = OnMessagesDeliveredFromRococoBulletin;
+	type MessageDispatch = XcmOverRococoBulletin;
+	type OnMessagesDelivered = XcmOverRococoBulletin;
 }
 
 /// Add support for the export and dispatch of XCM programs.
 pub type XcmOverPolkadotBulletinInstance = pallet_xcm_bridge_hub::Instance2;
 impl pallet_xcm_bridge_hub::Config<XcmOverPolkadotBulletinInstance> for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+
 	type UniversalLocation = UniversalLocation;
 	type BridgedNetwork = RococoBulletinGlobalConsensusNetworkLocation;
 	type BridgeMessagesPalletInstance = WithRococoBulletinMessagesInstance;
+
 	type MessageExportPrice = ();
 	type DestinationVersion =
 		XcmVersionOfDestAndRemoteBridge<PolkadotXcm, RococoBulletinGlobalConsensusNetworkLocation>;
-	type Lanes = ActiveLanes;
-	type LanesSupport = ToRococoBulletinXcmBlobHauler;
+
+	type AdminOrigin = EnsureRoot<AccountId>;
+	// Only allow calls from relay chains and sibling parachains to directly open the bridge.
+	type OpenBridgeOrigin = EnsureXcm<ParentRelayOrSiblingParachains>;
```

Comment:
This should be only `People` System Chain. We don't allow (yet) other (especially non-system) parachains to open bridge to Bulletin.

Reply:
good point, then we can even set it to `()` and disable it, 
because there is also another feature, where we can open channel with migration and for the bulletin it is already set it up: https://github.com/paritytech/polkadot-sdk/pull/4949/files#diff-e64ba9883926418979cf0f5886d108a30a96518d345b2ba819bcde75cbd8aa98R291-R298
```
	/// Ensure that the existing lanes for the People<>Bulletin bridge are correctly configured.
	pub type StaticToDynamicLanes = pallet_xcm_bridge_hub::migration::OpenBridgeForLane<
		Runtime,
		XcmOverPolkadotBulletinInstance,
		RococoPeopleToRococoBulletinMessagesLane,
		ConstBool<true>,
		PeopleRococoLocation,
		BulletinRococoLocation,
	>;
```

Reply:
actually, `OpenBridgeOrigin = ()` is not good, because People can also decide to close channel, I will set it for just People

Reply:
fixed here: https://github.com/paritytech/polkadot-sdk/pull/4949/commits/513aa6aeecdb7ebd7e8531cdca76ad8b36781f33